gen_cert.py

```diff
@@ -1901,38 +1910,42 @@ def fontlist_with_style(a_style):
 
         # SECTION: disclaimer text
         print_disclaimer = not self.cert_data.get('HAS_DISCLAIMER', False)
-        disclaimer_text = getattr(settings, 'CERTS_SITE_DISCLAIMER_TEXT', '')
-        if print_disclaimer and disclaimer_text:
+
+        if print_disclaimer and trans_disclaimer:
             (fonttag, fontfile, disclaimer_style) = font_for_string(
                 fontlist_with_style(style_small_text),
-                disclaimer_text,
+                trans_disclaimer,
```

Comment:
This (and above) diff disappears if we eliminate the `trans_` prefix and use the previous variable names.